src/js/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -44,121 +61,319 @@ export function formatDate(date) {
+        case, this may delay your appeal.</p>
+      );
+      break;
+    case STATUS_TYPES.bvaDevelopment:
+      contents.title = 'The judge is seeking additional information before making a decision';
+      contents.description = (
+        <p>The Board of Veterans’ Appeals is seeking evidence or an outside opinion from a legal,
+        medical, or other professional necessary to make decision about your appeal.</p>
+      );
+      break;
+    case STATUS_TYPES.stayed:
+      contents.title = 'The Board is waiting until a higher court makes a decision';
+      contents.description = (
+        <p>A higher court has requested that a group of appeals currently before the Board of
+        Veterans’ Appeals be held open. This is because the decision the court makes on a different
+        appeal could impact your appeal.</p>
+      );
+      break;
+    case STATUS_TYPES.remand: {
+      const { decisionIssues } = details;
+      const allowedIssues = decisionIssues
+        .filter((issue) => (issue.disposition === 'allowed'))
+        .map((issue, i) => (<li key={`allowed-${i}`}>{issue.description}</li>));
+      const deniedIssues = decisionIssues
+        .filter((issue) => (issue.disposition === 'denied'))
+        .map((issue, i) => (<li key={`denied-${i}`}>{issue.description}</li>));
+      const remandIssues = decisionIssues
+        .filter((issue) => (issue.disposition === 'remand'))
+        .map((issue, i) => (<li key={`remanded-${i}`}>{issue.description}</li>));
+
+      const pluralize = {
+        allowed: (allowedIssues.length > 1) ? 'these issues' : 'this issue',
+        denied: (deniedIssues.length > 1) ? 'issues' : 'issue',
+        remand: (remandIssues.length > 1) ? 'issues' : 'issue'
+      };
+
+      let allowedBlock = null;
+      let deniedBlock = null;
+      let remandBlock = null;
+      if (allowedIssues.length) {
+        allowedBlock = (
+          <div>
+            <h5 className="allowed-items">Allowed</h5>
+            <p>
+              The judge overrules the original decision and finds {pluralize.allowed} in
+              your favor
+            </p>
+            <ul>{allowedIssues}</ul>
+          </div>
+        );
+      }
+      if (deniedIssues.length) {
+        deniedBlock = (
+          <div>
+            <h5 className="denied-items">Denied</h5>
+            <p>The judge upholds the original decision for the following {pluralize.denied}</p>
+            <ul>{deniedIssues}</ul>
+          </div>
+        );
+      }
+      if (remandIssues.length) {
+        remandBlock = (
+          <div>
+            <h5 className="remand-items">Remand</h5>
+            <p>
+              The judge needs additional evidence to be collected or a procedural error to be
+              corrected for the following {pluralize.remand}
+            </p>
+            <ul>{remandIssues}</ul>
+            <p>The case is returned to the VBA to complete a remand order. </p>
```

Comment:
Take this line: The case is returned to the VBA to complete a remand order.

Reply:
I assume you mean take _out_ that line 😉 
Will do, could you please update the content in the [corresponding remand decision](https://github.com/department-of-veterans-affairs/vets.gov-team/blob/master/Products/Appeals%20Status/V2/Design%20files/Statuses.md#description-29) in the md?